src/screens/SendTokens.js

```diff
@@ -556,9 +603,29 @@ function SendTokens() {
       <div>
         <form ref={formSendTokensRef} id="formSendTokens">
           {renderOnePage()}
+          {renderDataOutputs()}
           <div className="mt-5">
-            <button type="button" className="btn btn-secondary mr-4" onClick={addAnotherToken}>{t`Add another token`}</button>
-            <button type="button" className="btn btn-hathor" onClick={onSendTokensClicked}>{t`Send Tokens`}</button>
+            <button
+              type="button"
+              className="btn btn-secondary mr-4"
+              onClick={addDataOutput}
+            >
+              {t`Add data output`}
+            </button>
+            <button
+              type="button"
+              className="btn btn-secondary mr-4"
+              onClick={addAnotherToken}
+            >
+              {t`Add another token`}
+            </button>
+            <button
+              type="button"
+              className="btn btn-hathor"
+              onClick={onSendTokensClicked}
+            >
+              {t`Send Tokens`}
+            </button>
```

Comment:
Since adding data outputs is an advanced feature should we have a checkbox for "Show advanced features" and only show the "Add data output" button when it is clicked?

If not, I think at least that the first option should be "Add another token" since it is the most commonly used, what do you think?

Reply:
I think we can have the button visible but with the add another token as the first option.

Reply:
I've changed the order.